fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -32,36 +32,42 @@ def self.get_build_number(params)
           build_nr = app.live_version.current_build_number
 
           UI.message("Latest upload for live-version #{app.live_version.version} is build: #{build_nr}")
+
+          return build_nr
         else
           version_number = params[:version]
 
-          # Get version number from latest pre-release if no version number given
+          # Used for determining how to display the optional version number
+          version_number_message = version_number.nil? ? "any version" : "version #{version_number}"
+
+          # Create filter for get_builds with optional version number
+          filter = { app: app.apple_id }
           if version_number
-            UI.message("Fetching the latest build number for version #{version_number}")
```

Comment:
we have two checks for `version_number.nil?` (here and line 41) could we consolidate?